File: src/Python/easy/array/problem1351.py
from typing import List

class Solution:
    def countNegatives(self, grid: List[List[int]]) -> int:
        count = 0
        for arr in grid:
            for item in arr:
                if item < 0:
                    count += 1
        return count



grid = [[4,3,2,-1],[3,2,1,-1],[1,1,-1,-2],[-1,-1,-2,-3]]

solution = Solution()
solution.countNegatives(grid)

File: src/JavaScript/easy/array/problem1351.js
/**
 * @param {number[][]} grid
 * @return {number}
 */

var countNegatives = function (grid) {
    let counter = 0
    for (let item of grid) {
        for (let i = 0; i < item.length; i++) {
            if (0 > item[i]) {
                counter++
            }
        }
    }
    return counter
};

let grid = [[4,3,2,-1],[3,2,1,-1],[1,1,-1,-2],[-1,-1,-2,-3]]

countNegatives(grid)